### ISSUE TITLE
Values not updated.

### ISSUE THREAD

**Issue by dvdmc** [SintefRaufossManufacturing/vc2opcua#2]
First, it must be noticed that I'm using KUKA.SIM. The server seems to work correctly since I can visualize all the data in the model. However, the values are not updated even when the simulation is started.

I'm sorry for opening an issue but I didn't find any contact information.

**Comment by alvcap** [comment]
Hi!

I have currently no access to VC nor KUKA.SIM, unfortunately I can’t try to reproduce your issue. 

Usually, in an OPCUA client, you can “subscribe” to data changes, that way the values of the nodes to which you are subscribed get updated every time they change. Have you tried that?

**Comment by AnajH** [comment]
Hi, 
could you find a solution for this problem? I have the same issue. Thanks

**Comment by mortlind** [comment]
@DavidMorillaCabello Did you check if you subscribed the client to the server, as @alvcap mentioned? This may be the same solution for you, @AnajH, so please check that.

**Comment by dvdmc** [comment]
Oh no, I wrote a message that was not sent. When I created the issue I was already using UAExpert to subscribe to the values but they don't change at any moment even when they clearly change.

**Comment by mortlind** [comment]
@DavidMorillaCabello I must admit that I have very close to zero knowledge about OPC-UA, I have never heard about UAExpert, and I have absolulte zero knowledge about VC. My best hope is that @alvcap is able to understand the problem, and find the time to reply. Otherwise, you are free to clone the project under GPL, and try to inspect and solve the issue yourself.

**Comment by dvdmc** [comment]
Thank you. I hope so too. In case that he can't solve the problem I will try to look for it myself. However, any help will be really useful.

**Comment by mortlind** [comment]
Just a bit of background: We work on a project basis at my company. This code was developed during a funded project, which is no longer active. As long as we do not have a new project depending on this code base, it is in "hibernation". Once we get a related project again, we have the funding to further develop the software, and we will then also have time available for looking into these issues. But until then ... :-(

**Comment by dvdmc** [comment]
Of course! I understand it. I posponed this specific project. However, as soon as I come back to it, I will try to help with this issue.

**Comment by mortlind** [comment]
Very good. We will be very grateful, if you push a merge request to us, if you find a solution!
